src/main/java/com/three_tech_solutions/slot_app/data/models/Student.java

```diff
@@ -30,8 +26,15 @@ public class Student {
     @OneToOne
```

Comment:
Siguiendo el comentario anterior, acá podríamos agregar lo siguiente
```suggestion
    @OneToOne(cascade = CascadeType.ALL)
```